native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/reflect-config.json

```diff
@@ -0,0 +1,271 @@
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setConnectTimeout",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "setFailOverReadOnly",
+        "parameterTypes": [
+          "boolean"
+        ]
+      },
+      {
+        "name": "setNoAccessToProcedureBodies",
+        "parameterTypes": [
+          "boolean"
+        ]
+      },
+      {
+        "name": "setPassword",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setQueriesBeforeRetryMaster",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "setSecondsBeforeRetryMaster",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "setSocketTimeout",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "setSslMode",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setTrustCertificateKeyStorePassword",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setTrustCertificateKeyStoreType",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setTrustCertificateKeyStoreUrl",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setUrl",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setUser",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      }
+    ]
+  },
+  {
+    "name": "com.mysql.cj.log.StandardLogger",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      }
+    ]
+  },
+  {
+    "name": "com.mysql.cj.protocol.StandardSocketFactory",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
+    "name": "io.netty.bootstrap.ServerBootstrap$1"
+  },
+  {
+    "name": "java.sql.Connection",
+    "methods": [
+      {
+        "name": "clearWarnings",
+        "parameterTypes": []
+      },
+      {
+        "name": "getAutoCommit",
+        "parameterTypes": []
+      },
+      {
+        "name": "getNetworkTimeout",
+        "parameterTypes": []
+      },
+      {
+        "name": "getTransactionIsolation",
+        "parameterTypes": []
+      },
+      {
+        "name": "isReadOnly",
+        "parameterTypes": []
+      },
+      {
+        "name": "isValid",
+        "parameterTypes": [
+          "int"
+        ]
+      },
+      {
+        "name": "setNetworkTimeout",
+        "parameterTypes": [
+          "java.util.concurrent.Executor",
+          "int"
+        ]
+      }
+    ]
+  },
+  {
+    "name": "org.quartz.simpl.CascadingClassLoadHelper",
```

Comment:
quartz related configs are already added through task module. So can we remove this and check?
Also check other quartz related classes

Reply:
Removed in [9f68040](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/653/commits/9f68040d7ddf069c53730192d708bfa74bc74e81)